notebooks/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -0,0 +1,817 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# LightGBM: A Highly Efficient Gradient Boosting Decision Tree\n",
+    "This notebook will give you a quick example of how to train LightGBM model in a recommendation scenario. \n",
+    "LightGBM \\[1\\] is a gradient boosting framework that uses tree-based learning algorithms. It is designed to be distributed and efficient with the following advantages:\n",
+    "* Faster training speed and higher efficiency.\n",
```

Comment:
"Faster" / "lower" / "better" are used without comparing LightGBM to anything. Can we be more clear? IE "better accuracy and faster training speed than _____"?